Furball.Vixie.Backends.OpenGL41/OpenGL41Backend.cs

```diff
@@ -97,10 +99,18 @@ public class OpenGL41Backend : IGraphicsBackend, IGLBasedBackend {
             
             BackendInfoSection mainSection = new BackendInfoSection("OpenGL Info");
             mainSection.Contents.Add(("OpenGL Version", this.gl.GetStringS(StringName.Version)));
+            this.CheckError("check verison");
             mainSection.Contents.Add(("GLSL Version", this.gl.GetStringS(StringName.ShadingLanguageVersion)));
+            this.CheckError("check glsl version");
             mainSection.Contents.Add(("OpenGL Vendor", this.gl.GetStringS(StringName.Vendor)));
+            this.CheckError("check check vendor");
             mainSection.Contents.Add(("Renderer", this.gl.GetStringS(StringName.Renderer)));
-            // mainSection.Contents.Add(("Supported Extensions", this.gl.GetStringS(StringName.Extensions)));
+            this.CheckError("check check renderer");
```

Comment:
are all of those 50 billion checkerrors really necessary, especially on something like GetString? those feel REALLY excessive,

Reply:
given that i added them *because* i was getting a gl error from them and that they only get run on debug, its better to be safe then sorry